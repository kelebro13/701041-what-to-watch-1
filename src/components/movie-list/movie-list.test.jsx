import renderer from "react-test-renderer";
import MovieList from "./movie-list";

it(`render correctly MovieList component`, () => {
  const films = [
    {
      id: 1,
      name: `The Grand Budapest Hotel`,
      previewImage: `img/the-grand-budapest-hotel.jpg`,
      previewVideoLink: `https://some-link`,
      genre: `Comedy`,
    }
  ];

  const tree = renderer
    .create(<MovieList films={films} activeCard={-1} onActiveCardChange={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
