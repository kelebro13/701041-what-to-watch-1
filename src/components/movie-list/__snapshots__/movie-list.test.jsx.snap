// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly MovieList component 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        muted={true}
        poster="img/the-grand-budapest-hotel.jpg"
        src="https://some-link"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        The Grand Budapest Hotel
      </a>
    </h3>
  </article>
</div>
`;
